exporters/filters/dupe_filter.py

```diff
@@ -0,0 +1,29 @@
+from exporters.filters.base_filter import BaseFilter
+
+
+class DupeFilter(BaseFilter):
+    """
+    Filter items depending on a key field.
+
+        - key_field (str)
+            item's key to be used to identify dupes
+    """
+    # List of options
+    supported_options = {
+        'key_field': {'type': basestring, 'default': '_key'}
+    }
+
+    def __init__(self, *args, **kwargs):
+        super(DupeFilter, self).__init__(*args, **kwargs)
+        self.key_field = self.read_option('key_field')
+        self.key_set = set()
+
+    def filter(self, item):
+        items_key = item[self.key_field]
```

Comment:
use `item.get(self.key_field)`, otherwise it will raise a `KeyError` when key field isn't present